frontend/src/components/project/Buttons/GoBackButton.js

```diff
@@ -0,0 +1,56 @@
+import { Button, IconButton, makeStyles } from "@material-ui/core";
+import ArrowBackIcon from "@material-ui/icons/ArrowBack";
+import React from "react";
+import ArrowBackIosIcon from "@material-ui/icons/ArrowBackIos";
+import { useRouter } from "next/router";
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    color: "white",
+    [theme.breakpoints.down("xs")]: {
+      minWidth: 35,
+      maxWidth: 35,
+      minHeight: 35,
+      maxHeight: 35,
+      backgroundColor: theme.palette.primary.main,
+      "&:hover": {
+        backgroundColor: theme.palette.primary.main,
+      },
+    },
+  },
+}));
+
+export default function GoBackButton({ texts, tinyScreen, locale, containerClassName }) {
+  const classes = useStyles();
+
+  const router = useRouter();
```

Comment:
The goBack-function only works for the projects page right now. Should I generalize it?

Reply:
Good point. I think it's quite hard to generalize it unless you pass the page you want to go back to manually. Therefore it would maybe be better to rename it so that it's clear that it's not generic. E.g. `GoBackFromProjectPageButton`